buddhi/README.md

```diff
@@ -37,9 +37,26 @@ $ curl http://127.0.0.1:8089/paths/backend?lines=1
 ```bash
 $ curl http://127.0.0.1:8089/paths/amp?lines=1
 "30dd63c1-4ee9-481e-b0ed-e7d10d4900c9","/1?user_key=2c6e8625ed43d064"
+```
+ - POST **/report/amp**: Send traffic file and generate metric counter report.
+```bash
+$ curl -X POST --data-binary "@traffic.csv" http://127.0.0.1:8089/report/amp 2>/dev/null | jq '.'
+{
+  "metrics": {
+    "24b08f30-403e-4be7-83d8-984d6e93b91a": 7,
```

Comment:
A metric ID by itself does not identify a metric. It also needs the service ID.
If I remember correctly, Apisonator will let you create 2 different metrics with the same ID under different services.

Reply:
Correct,

What format would be appropriate?

```
{
   "svc_a":{
       "metric_a_1": 3,
       "metric_a_2": 6,
        "metric_a_3": 8,
       }
   },
   "svc_b":{
       "metric_b_1": 1,
       "metric_b_2": 2,
        "metric_b_3": 7,
       }
   }
}
```

or CSV

```
"service", "metric", "counter"
"svc_a", "metric_a_1", 5
"svc_a", "metric_a_2", 5
"svc_a", "metric_a_3", 5
"svc_b", "metric_b_1", 5
"svc_b", "metric_b_2", 5
"svc_b", "metric_b_3", 5
```

Reply:
I'd return JSON in this case because it allows us to represent the `service -> metric` relationships